File: 백준/Bronze/5086. 배수와 약수/배수와 약수.cc
 #include <iostream>

int main() {
  while (true) {
    int n, m;

    std::cin >> n >> m;
    if (n == 0 && m == 0) {
      break;
    } else if (m % n == 0) {
      std::cout << "factor" << std::endl;
    } else if (n % m == 0) {
      std::cout << "multiple" << std::endl;
    } else {
      std::cout << "neither" << std::endl;
    }
  }
}

File: 백준/Bronze/5086. 배수와 약수/배수와 약수.py
import sys

def main() -> None:
    outputs = []
    for line in sys.stdin:
        a, b = map(int, line.split())
        if a == 0 and b == 0:
            break
        if b % a == 0:
            outputs.append('factor')
        elif a % b == 0:
            outputs.append('multiple')
        else:
            outputs.append('neither')
    sys.stdout.write('
'.join(outputs))

if __name__ == '__main__':
    main()
